scripts/exglobal_aero_analysis_finalize.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# exgdas_global_aero_analysis_finalize.py
+# exglobal_aero_analysis_finalize.py
```

Comment:
should all of these 9 files that have the comments changed be in this PR or separate? @aerorahul @WalterKolczynski-NOAA

Reply:
that's fine.